packages/core/docs/advanced/calling-platform-api.md

```diff
@@ -0,0 +1,28 @@
+# Calling platform API
+
+The Vue Storefront has its own way to communicate with other platform APIs. First of all, each integration implements an API-client that defines an interaction and connection with a given platform. Each time you make a call to the external platform, you actually use API-client beneath. We use that in all of the integrations and also you can reach the API-client in your project if it's needed.
```

Comment:
1. This paragraph provides 0 knowledge about API client works and what it is
2. Does user even need this information?